packages/server/graphql/public/mutations/helpers/publishTeamPromptMentions.ts

```diff
@@ -0,0 +1,54 @@
+import {JSONContent} from '@tiptap/core'
+import getRethink from '../../../../database/rethinkDriver'
+import NotificationResponseMentioned from '../../../../database/types/NotificationResponseMentioned'
+import {TeamPromptResponse} from '../../../../postgres/queries/getTeamPromptResponsesByIds'
+
+const getMentionedUserIdsFromContent = (content: JSONContent): string[] => {
+  if (content.type === 'mention' && content.attrs?.id) {
+    return [content.attrs?.id]
```

Comment:
+1 can remove optional chaining operator, the conditional above already narrowed the type